src/component/mxgraph/config/StyleConfigurator.ts

```diff
@@ -84,22 +84,22 @@ export default class StyleConfigurator {
     [
       AssociationDirectionKind.NONE,
       (style: StyleMap) => {
-        style[mxConstants.STYLE_STARTARROW] = null;
-        style[mxConstants.STYLE_ENDARROW] = null;
-        style[mxConstants.STYLE_EDGE] = null; // ensure no orthogonal segments, see also https://github.com/process-analytics/bpmn-visualization-js/issues/295
+        style[mxConstants.STYLE_STARTARROW] = undefined;
```

Comment:
:information_source: null --> undefined changed because the way we check this style property has changed in model tests